cmd/cluster/kubevirt/create.go

```diff
@@ -45,6 +45,7 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.RootVolumeStorageClass, "root-volume-storage-class", opts.KubevirtPlatform.RootVolumeStorageClass, "The storage class to use for machines in the NodePool")
 	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.RootVolumeSize, "root-volume-size", opts.KubevirtPlatform.RootVolumeSize, "The size of the root volume for machines in the NodePool in Gi")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.RootVolumeAccessModes, "root-volume-access-modes", opts.KubevirtPlatform.RootVolumeAccessModes, "The access modes of the root volume to use for machines in the NodePool (comma-delimited list)")
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.RootVolumeVolumeMode, "root-volume-volume-mode", opts.KubevirtPlatform.RootVolumeVolumeMode, "The volume mode of the root volume to use for machines in the NodePool. supported values are \"Block\", \"Filesystem\"")
```

Comment:
There is no validation on that.
A user can put something other than `Block` or `Filesystem` and no error will be shown, just the arbitrary value won't persist in in the nodepool object.
on the other hand, if putting an invalid value on the PVC resource, the request got rejected.

Reply:
Also, "supported" should be capitalize since it's a new sentence.

Reply:
This is blocking merges in the hypershift repo.  Lets track this as a follow on an merge this to get things back on the rails.